modules/cats/src/main/tut/README.md

```diff
@@ -39,3 +42,38 @@ then load the config:
 ```tut:book
 loadConfig[MyVecConfig](conf)
 ```
+
+### Using cats type class instances for readers and writers
+
+In order to to put in scope the `cats` type classes for our readers and writers and extend the latter with the extra
+operations provided by `cats`, we need some extra imports:
+
+```tut:silent
+import cats._
+import cats.implicits._
+import pureconfig.module.cats.instances._
+import pureconfig.syntax._
```

Comment:
You're right in all of them. I replaced `cats.implicits._` by `cats.syntax.all._` and replaced the `toConfig` by the `writer.to(...)` syntax.